tupelo/repotree/repotree.go

```diff
@@ -11,7 +11,7 @@ import (
 	tupelo "github.com/quorumcontrol/tupelo-go-sdk/gossip/client"
 )
 
-var repoSalt = []byte("decentragit-alpha")
+var repoSalt = []byte("decentragit-0.0.0-alpha")
```

Comment:
just changed this to "reset" the universe of dgit chaintrees since now we actual user keys